.travis.yml

```diff
@@ -118,11 +118,6 @@ deploy:
   provider: releases
   skip_cleanup: true
 
-cache: cargo
-before_cache:
```

Comment:
Lets replace this with a comment explaining why we cannot use caching right now, and when we should be able to re-enable it